packages/gatsby-cli/src/create-cli.js

```diff
@@ -26,6 +31,21 @@ function buildLocalCommands(cli, isLocalSite) {
     siteInfo.browserslist = json.browserslist || siteInfo.browserslist
   }
 
+  function installedGatsbyVersion() {
+    let majorVersion
+    try {
+      let packagePath = require.resolve(
+        path.join(process.cwd(), `node_modules`, `gatsby`, `package.json`)
+      )
+      const packageInfo = JSON.parse(fs.readFileSync(packagePath))
```

Comment:
You can do ``require(`gatsby/package.json`)`` like in here I think:
https://github.com/gatsbyjs/gatsby/blob/bc1a79de577ac9f2c77eaac3bcaa4a8368e3ad35/packages/gatsby/cache-dir/static-entry.js#L11

Reply:
Oh yeah, straight up require. Thanks 😅 

I _think_ this still needs to use `process.cwd()` to provide an absolute path - otherwise the require doesn't resolve the local `node_modules` directory from a globally installed `gatsby-cli`. Maybe there's a neater way?